src/components/Navbar/UserMenu.js

```diff
@@ -100,10 +100,8 @@ const UserMenuContent = React.forwardRef(({ open, isMobile = false, sessionData,
                 autoFocusItem={open}
                 id="menu-list-grow"
             >
-                {/*
-                <MenuItem button disableTouchRipple onClick={() => {}}>My Offers</MenuItem>     // These options need to be implemented
+                <MenuItem button component={Link} to="/company/offers/manage" disableTouchRipple>My Offers</MenuItem>
```

Comment:
This should only show up to company users, similar to what happens with AdminMenuOptions